mitodl/ocw-to-hugo
# popup video links

#### Pre-Flight checklist


#### What are the relevant tickets?
https://github.com/mitodl/ocw-to-hugo/issues/235

#### What's this PR do?
Recently the `template_type` property was added to items in `course_embedded_media` in OCW parsed JSON.  This PR looks for items with a `template_type` of `popup` and instead of writing in an embedded video, a link to the single video player page is written in.

#### How should this be manually tested?
 - Read the readme if you have not used this utility before
 - Generate markdown for a course that has popup type videos, such as `18-03sc-differential-equations-fall-2011` or `21g-107-chinese-i-streamlined-fall-2014`
 - Verify that popup type videos have a link rendered instead of an embedded video, and that the link leads to the page for that video

Some examples of pages with popup type videos in `18-03sc-differential-equations-fall-2011` can be found in `content/sections/unit-i-first-order-differential-equations/solution-by-integrating-factors/`
